README.md

```diff
@@ -30,89 +81,164 @@ $response = $fraudService->validateRequest($request);
 
-foreach ($response->getMessages() as $message) {
-    if ($message->getType() !== \Omnifraud\Contracts\MessageInterface::TYPE_INFO) {
-        $this->saveOrderMessage($message->getType(), $message->getMessage());
-    }
-}
+## Fraud Services / Drivers
 
-```
-Note: See [MakesTestRequest@makeTestRequest()](https://github.com/lxrco/omnifraud-common/blob/master/src/Testing/MakesTestRequests.php) for a full example of a request, each service might require different fields but they can all handle a full request.
+All drivers implement [ServiceInterface](https://github.com/lxrco/omnifraud-common/blob/master/src/Contracts/ServiceInterface.php).
 
-## Installation
+The following services are officially supported right now:
 
-Usually all you need to do is installing the service you need, for example:
-```bash
-composer require omnifraud/signifyd
+Service | Composer Package | Alias | Maintainer
+--- | --- | --- | ---
+[Kount](https://github.com/lxrco/omnifraud-kount) | omnifraud/kount | Kount | [LXRandCo](https://github.com/lxrco)
+[Signifyd](https://github.com/lxrco/omnifraud-signifyd) | omnifraud/signifyd | Signifyd | [LXRandCo](https://github.com/lxrco)
+[Null](https://github.com/lxrco/omnifraud-common) | omnifraud/common | Null | [LXRandCo](https://github.com/lxrco)
+
+> *Note: Interested in contributing your own implementation? We'd love to include it! Add it to the above list and send a PR.*
+
+### The `Null` Driver
+
+The `Null` driver does nothing. What else were you expecting?
+
+## Usage
+
+The lifecycle of the fraud request looks like this:
+
+1. User visits www.example.com
+2. User is assigned a random session ID or reuses existing session ID (via local storage, cookies, etc.)
+3. User attempts checkout with payment
+4. Both failed and successful payments are reported to fraud service
+5. Once checkout succeeds, a score is given to the fraud request (either synchronously or asynchronously) and is saved alongside the order
+
+In order of implementation, this translates into:
+
+1. Instantiation of a driver
+2. Front-end JavaScript implementation
+3. Creation of a fraud "request"
+4. On checkout, recording *order*, *session*, *account*, and *payment* info in the request
+5. Storing the response and fraud score in the database, or queuing for later retrieval if asynchronous
+
+### The Session ID
+
+It's entirely up to you to generate a session ID and re-use the same one throughout the aforementioned fraud request lifecycle. Most services have no opinion on what it looks like, but to be safe accross most vendors you should ensure it:
+
+* is alphanumeric
+* is of reasonable length (more than 255 might be overkill)
+* has some uniqueness properties (tip: don't do things like `crc32(time())`)
+
+Once you have your random session ID, toss it into a cookie (or custom header, however you want to communicate it) and use it for both the front-end tracker and the back-end fraud requests.
```

Comment:
Not sure what using a header here would look like. The most important part is that it does not change between page load, and is the same on client and server.